doc/changes/latest.inc

```diff
@@ -50,6 +53,8 @@ Enhancements
 
 - Further improved documentation building instructions and execution on Windows (:gh:`8502` by `kalenkovich`_ and `Eric Larson`_)
 
+- Add :class:`mne.decoding.SSD` for spatial filtering with spatio-spectral-decomposition (:gh:`7070` **by new contributor** |Victoria Peterson|_ and `Denis Engemann`_)
```

Comment:
New contributors at the top of the given section

https://23402-1301584-gh.circle-artifacts.com/0/dev/install/contributing.html#first-time-contributors

Reply:
shall now we also change this @dengemann  ?

Reply:
@dengemann already pushed a commit!